vars/uploadMinSnapshotsToS3.groovy

```diff
@@ -46,47 +46,51 @@ void call(Map args = [:]) {
     String dstDir = "snapshots/core/${productName}/${version}"
     String baseName = "${productName}-min-${version}-${platform}-${architecture}"
     argsMap = [:]
+    argsMap['sigtype'] = '.sig'
 
-    // Setup core plugins (Tar x64 only)
-    String corePluginDir = "${WORKSPACE}/${distribution}/builds/${productName}/core-plugins".replace("\\", "/")
-    boolean corePluginDirExists = fileExists(corePluginDir)
-    if (architecture == "x64" && platform == "linux" && distribution == "tar" && corePluginDirExists) {
-        echo("Create .sha512 for Core Plugins Snapshots")
-        argsMap['artifactPath'] = corePluginDir
-        for (Closure action : fileActions) { // running createSha512Checksums()
-            action(argsMap)
-        }
-    }
-
-    // Create checksums
-    echo('Create .sha512 for Min Snapshots Artifacts')
-    argsMap['artifactPath'] = srcDir
-    for (Closure action : fileActions) { // running createSha512Checksums()
-        action(argsMap)
-    }
-
-    echo("Start copying files: version-${version} revision-${revision} architecture-${architecture} platform-${platform} buildid-${id} distribution-${distribution} extension-${extension}")
-
-    String sedCmd = "sed"
-    if (platform == "darwin") {
-        sedCmd = "gsed"
-    }
-
-    sh """
-        cp -v ${srcDir}/${baseName}.${extension} ${srcDir}/${baseName}-latest.${extension}
-        cp -v ${srcDir}/${baseName}.${extension}.sha512 ${srcDir}/${baseName}-latest.${extension}.sha512
-        cp -v ${srcDir}/../manifest.yml ${srcDir}/${baseName}-latest.${extension}.build-manifest.yml
-        ${sedCmd} -i "s/.${extension}/-latest.${extension}/g" ${srcDir}/${baseName}-latest.${extension}.sha512
-    """
     withCredentials([
         string(credentialsId: 'jenkins-artifact-promotion-role', variable: 'ARTIFACT_PROMOTION_ROLE_NAME'),
         string(credentialsId: 'jenkins-aws-production-account', variable: 'AWS_ACCOUNT_ARTIFACT'),
         string(credentialsId: 'jenkins-artifact-production-bucket-name', variable: 'ARTIFACT_PRODUCTION_BUCKET_NAME')]) {
+
```

Comment:
Any reason we move this section of codes within the credential role along with uploading/adding the signature?